# Example 4.1

Using the tube analogy what will be the capillary rise in a uniform loose sand with $d=0.15 \mathrm{mm}$ and in a clay with $D_{10}=1.5\mu \mathrm{m}$. Assume the pore size is 20% of $D_{10}$.

## Data:

$d=0.15 \mathrm{mm}$

$D_{10}=1.5\mu \mathrm{m}$

$T=0.07275 \mathrm{N/m}$ Water surface tension at 20 deg C

## Principles

$h_c=-\cfrac{4T}{\gamma_w d_t}$

$d_t \approx 0.2 D_{10}$

In [4]:
import numpy as np

In [5]:
# I first put everything in consistent units
d_clay=1.5e-6 #clay D10 in m
d_sand=1.5e-4 #sand particle size in m
dt=[d_sand, 0.2*d_clay] #I only use 0.2 factor for the clay because the sand is uniform particles

In [10]:
T=0.07275/1000 # water tension in kN/m
hc=-4*T/(9.81*np.array(dt)) #capillary rise
print("Sand hc= %1.2f m" %hc[0])
print("Clay hc= %1.2f m" %hc[1])


Sand hc= -0.20 m
Clay hc= -98.88 m


# Example 4.2

Calculate the capillarity water pressure for the same soils of Example 4.1.

## Data:

Clay $h_c=98.88$ m

Sand $h_c=0.20$ m

## Principles

$u_c=-h_c \gamma_w$

In [12]:
uc=hc*9.81 # calculates the pressure
print("Sand uc= %1.2f kPa" %uc[0])
print("Clay uc= %1.2f kPa" %uc[1])

Sand uc= -1.94 kPa
Clay uc= -970.00 kPa
